docs/introduction/what-is-cdf.md

```diff
@@ -0,0 +1,30 @@
+# What is CDF
+
+OpenFABR CDF aims to offer a brand new way of building cloud infrastructure, based on our view that the future of DevOps is **MODULAR + CUSTOM**.
+
+![CDF: MODULAR + CUSTOM](../assets/cdf-module_custom.png)
+
+We envisage a future where application developers (in product engineering) and DevOps engineers (in platform engineering) collaborate by code in a true Infrastructure-as-Code manner. There are various IaC runtime, imperative language supports and existing software delivery infrastructure such as code and artefact repositories. OpenFABR CDF is the missing piece that makes it happen.
```

Comment:
Not sure what this is saying. 
>There are various IaC runtime, imperative language supports and existing software delivery infrastructure such as code and artefact repositories.

Reply:
This diagram ? 
<img width="572" alt="image" src="https://user-images.githubusercontent.com/51779/213171158-40dc8780-6715-4512-ac61-ff5c772027a0.png">

Reply:
```suggestion
We envisage a future where application developers (in product engineering) and DevOps/Cloud engineers (in platform engineering) collaborate by code in a true Infrastructure-as-Code manner. OpenFABR CDF is the missing piece, the glue, that brings the entire tool chain together from IaC runtimes like CDK, to imperative languages like TypeScript, testing frameworks like Jest, and artefact repositories like NPM.
```

Reply:
how does that^ sound?